### ISSUE TITLE
UPPERCASE and underlined names are very difficult to read

### ISSUE THREAD

**Issue by jancborchardt** [libasys/contactsplus#23]
Because people perceive words by their form as well, capitalizing something makes it very hard to interpret. Contact names should _not_ be capitalized. Also the underline is very ugly and should be removed.

**Comment by libasys** [comment]
please don't use words such ugly etc this is a personal feeling! Thanks

**Comment by jancborchardt** [comment]
Ok, sorry about that. Let me rephrase: The underline is not used anywhere else in ownCloud. Visually and typographically it does not look appealing either, so it should be removed.

**Comment by libasys** [comment]
@jancborchardt the rework of the contacts+ app becomes starting after finishing the rework of the calendar app, so i will read your posted issues and on the rework i will follow up the design guide of OC!

**Comment by jancborchardt** [comment]
@libasys great, thanks! :) Keep in mind to not have a giant rework branch though … small steps, small pull requests. Otherwise it will be the same story again and someone might fork _your_ app then. ;)
